pkg/compactor/compactor.go

```diff
@@ -213,8 +238,8 @@ type Config struct {
 	ShardingStrategy string     `yaml:"sharding_strategy"`
 	ShardingRing     RingConfig `yaml:"sharding_ring"`
 
-	// Compaction mode.
-	CompactionStrategy string `yaml:"compaction_mode"`
+	// Compaction strategy.
+	CompactionStrategy string `yaml:"compaction_strategy"`
```

Comment:
just to confirm, there is no breaking change here, correct? this was added recently and the partition was not implemented yet.

Reply:
That is right.